pkg/base/base_test.go

```diff
@@ -120,10 +123,11 @@ spec:
 			assert.Equal(t, test.expected, string(b.Content))
 		})
 	}
-
 }
 
 func Test_ShouldBeIncludedInBaseKustomization(t *testing.T) {
+	test := scopeagent.StartTest(t)
+	defer test.End()
```

Comment:
this is also instrumented within the test case loop

Reply:
thanks, fixed